packages/core/src/extension/host-origin-event.ts

```diff
@@ -0,0 +1,4 @@
+// TODO - will finalize in the following PR
```

Comment:
Maybe just remove it for now because it's not used. We can review how it gets used in the following PR?

Reply:
It's used in the post method of `extensionMessenger` as that method posts messages to an extension.
```
post(extensionId: string, event: HostOriginEvent): void {
```